cmd/webhook/server/readiness_probe.go

```diff
@@ -0,0 +1,48 @@
+package server
+
+import (
+	"fmt"
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog"
+	"net/http"
+)
+
+const CRDsAmount = 8
+
+type ReadinessCRD struct {
+	client  *apiextensionsclientset.Clientset
+	release string
+}
+
+// NewReadinessCRDProbe returns pointer to ReadinessCRD
+func NewReadinessCRDProbe(config *rest.Config, releaseName string) (*ReadinessCRD, error) {
+	apiextensionsClient, err := apiextensionsclientset.NewForConfig(config)
+	if err != nil {
+		return &ReadinessCRD{}, err
+	}
+	return &ReadinessCRD{apiextensionsClient, releaseName}, nil
+}
+
+// Name returns name of readiness probe
+func (r ReadinessCRD) Name() string {
+	return "ready-CRDs"
+}
+
+// Check if all CRDs with specific label are ready
+func (r *ReadinessCRD) Check(_ *http.Request) error {
```

Comment:
Discussed f2f

- please move that logic under the `/pkg` directory
- please check the status of the CRD (if it's accepted by the API Server)
- please use that approach both in webhook server and controller-manager